planner/memo/group.go

```diff
@@ -87,15 +88,40 @@ type Group struct {
 	EngineType EngineType
 }
 
+// buildLogicalProperty builds a new LogicalProperty for the new GroupExpr.
+// it also builds the unique key info and max one row info.
+func buildLogicalProperty(e *GroupExpr, oldSchema *expression.Schema) *property.LogicalProperty {
+	prop := &property.LogicalProperty{}
+	if oldSchema == nil {
```

Comment:
What is this branch for?

Reply:
I found it would panic here, so I add this check. But I just figure it out. It is because some unit test cases use `NewGroupWithSchema(expr, nil)`. I'll remove this check and modify the tests.